pkg/csi/cinder/controllerserver.go

```diff
@@ -261,6 +261,13 @@ func (cs *controllerServer) ListVolumes(ctx context.Context, req *csi.ListVolume
 				CapacityBytes: int64(v.Size * 1024 * 1024 * 1024),
 			},
 		}
+
+		status := &csi.ListVolumesResponse_VolumeStatus{}
+		for _, attachment := range v.Attachments {
```

Comment:
just curious , this attachments means we considered multi attach feature?
one volume attached to multiple node?

Reply:
I also think normally block storage is attached to one node.
But cinder response supports multiple attachments in API level.
https://docs.openstack.org/api-ref/block-storage/v3/index.html?expanded=list-accessible-volumes-with-details-detail#list-accessible-volumes-with-details
and actually some driver may support multi-attach:
https://docs.openstack.org/cinder/latest/admin/blockstorage-volume-multiattach.html